src/components/Congress/Overview/index.js

```diff
@@ -29,6 +33,14 @@ export default function Overview() {
           Apply for Congress
         </Button>
       )}
+      <Button
```

Comment:
This button should only be shown to congress members.

Reply:
blocked by https://github.com/liberland/liberland_frontend/pull/115/files#diff-4bbf5fe67dacc75c32883c4a98cfdd9de61b437adf97f0ac410db1b4c3e9b0fe